recipes-wpe/wpewebkit/wpewebkit_0.1.bb

```diff
@@ -52,6 +52,7 @@ PACKAGECONFIG[westeros-mesa] = "-DUSE_WPEWEBKIT_BACKEND_WESTEROS_MESA=ON,,"
 # WPE Platform specific switches
 PACKAGECONFIG[intelce] = "-DUSE_WPEWEBKIT_PLATFORM_INTEL_CE=ON -DUSE_HOLE_PUNCH_GSTREAMER=ON,,"
 PACKAGECONFIG[nexus] = "-DUSE_WPEWEBKIT_PLATFORM_BCM_NEXUS=ON -DUSE_HOLE_PUNCH_GSTREAMER=ON,,"
+PACKAGECONFIG[qcomdb] = "-DUSE_WPEWEBKIT_PLATFORM_DB410C=ON,,"
```

Comment:
Currently only DB410C is using this switch and I hope the same can be used for similar board DB820c also in the future.